examples/translate_circle.ml

```diff
@@ -0,0 +1 @@
+  let c = circle 100 in
```

Comment:
@nangahamandine I think you accidentally deleted the rest of the file.

Reply:
What @Sudha247 meant was that in the original example...

(original example)
```ocaml
let () =
  init ();
  set_color black;

  (* Create circle transform *)
  let c = circle 100 in

  (* Display circle transform *)
  show [c];

  close ()
```

you didn't actually apply `translate` to the circle before displaying it.

The expected code is something like this instead:
```ocaml
let () =
  init ();
  set_color black;

  (* Create circle *)
  let c1 = circle 100 in
  (* Translate it to the right by 100 *)
  let c2 = translate 100 0 c1 in
  (* Display both circles *)
  show [c1; c2];

  close ()
```